WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -0,0 +1,742 @@
+import Foundation
+import WordPressComAnalytics
+import WordPressComStatsiOS
+import WordPressShared
+
+@objc class PostListViewController : AbstractPostListViewController, UIViewControllerRestoration, PostCardTableViewCellDelegate {
+    
+    static private let postCardTextCellIdentifier = "PostCardTextCellIdentifier"
+    static private let postCardImageCellIdentifier = "PostCardImageCellIdentifier"
+    static private let postCardRestoreCellIdentifier = "PostCardRestoreCellIdentifier"
+    static private let postCardTextCellNibName = "PostCardTextCell"
+    static private let postCardImageCellNibName = "PostCardImageCell"
+    static private let postCardRestoreCellNibName = "RestorePostTableViewCell"
+    static private let postsViewControllerRestorationKey = "PostsViewControllerRestorationKey"
+    static private let statsStoryboardName = "SiteStats"
+    static private let currentPostListStatusFilterKey = "CurrentPostListStatusFilterKey"
+    static private let currentPostAuthorFilterKey = "CurrentPostAuthorFilterKey"
```

Comment:
Yep - that's one of the improvements that can be done.  I avoided these kinds of things in this particular PR on purpose to make sure I'm not introducing new bugs and to make sure the code is easier to compare for the reviewers.